app/controllers/user_controller.rb

```diff
@@ -537,6 +538,15 @@ def current_user_is_display_user
     @user.try(:id) == @display_user.id
   end
 
+  def show_about_me?
```

Comment:
Code Climate is unhappy with the complexity and number of returns here. I somewhat agree, but this felt more readable than a big combination of `||`, `&&` and `!`s.

I should add a comment to the method that these statements are order-dependent.

Reply:
If you change the code climate warning to wontfix then this PR will go green

Reply:
Yeah I know that – It's more that it's a valid criticism so wanted to sense-check what you thought and whether there's a nicer alternative approach.